### PR TITLE
Rf-294 Remove Atomize Button (re-opening to merge into main)

### LINKED ISSUE #294: [Refactor] Get rid of atomzie-wrapper Button

### PR DESCRIPTION
**Pull request by siberowl**
## Which Issue was this PR made for?

- Closes #294 

## What is within the scope with this PR

Merging it into main.

### CODE REVIEW

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-333.dev.mochatto.com

File: client/src/pages/JoinPage/index.tsx [hunk 1]
```diff
@@ -105,6 +107,14 @@ const JoinPage = ({
 
   useEffect(() => {
     setVisualizer(new AudioVisualizer(onAudioActivity));
+    const keyListener = (event) => {
+      console.log(event);
```
Comment (Njima1572) [inline client/src/pages/JoinPage/index.tsx:111]: console.log

File: client/src/templates/style.scss [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.close-button {
+ color: rgb(80, 82, 86);
+ background: rgb(247, 248, 249);
```
Comment (Njima1572) [inline client/src/templates/style.scss:3]: Not a fan of having this class here. What do you think about moving this in `components/Button/style.scss`, or change to `secondary` button

File: client/src/pages/JoinPage/index.tsx (hunk 1 above)
Comment (Njima1572) [inline client/src/pages/JoinPage/index.tsx:111]: console.log

File: client/src/components/ButtonsBar/style.scss [hunk 3]
```diff
@@ -1,4 +1,35 @@
...
+  justify-content: center;
+
+  .buttons-bar {
+    background: rgba(0, 0, 0, 0.73);
+    padding: 0.3rem 1rem;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-evenly;
+    align-items: center;
+    border-radius: 5rem;
```
Comment (Njima1572) [inline client/src/components/ButtonsBar/style.scss:20]: Would be nice to have default border-radius or something

File: client/src/components/Button/style.scss [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+}
+
+
+button:hover {
+  background-color: rgb(222, 222, 222, 0.2);
+}
```
Comment (Njima1572) [inline client/src/components/Button/style.scss:26]: Could move into `button{}` to match the style with other ones.

**Review by Njima1572 — CHANGES_REQUESTED** [review]
Mostly looking great!

File: client/src/templates/style.scss (hunk 2 above)
Comment (siberowl) [reply client/src/templates/style.scss:3]: I agree yeah. I just moved it into the RoomTemplate

File: client/src/components/ButtonsBar/style.scss (hunk 3 above)
Comment (siberowl) [reply client/src/components/ButtonsBar/style.scss:20]: That's true I'll move it into the constants

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-333.dev.mochatto.com

**Comment by Njima1572** [comment]
AUTOMATED: Deployed to https://pr-333.dev.mochatto.com

**Review by Njima1572 — APPROVED** [review]
LGTM

**Pull request merged**